apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/PathAssertionsSpec.kt

```diff
@@ -29,6 +32,14 @@ class PathAssertionsSpec : ch.tutteli.atrium.specs.integration.PathAssertionsSpe
         private fun isWritable(expect: Expect<Path>) = expect toBe writable
         private fun isRegularFile(expect: Expect<Path>) = expect toBe aRegularFile
         private fun isDirectory(expect: Expect<Path>) = expect toBe aDirectory
+        @Suppress("UNUSED_PARAMETER")
+        private fun hasSameTextualContentAs(expect: Expect<Path>, targetPath: Path, sourceCharset: Charset, targetCharset: Charset): Expect<Path> {
+            return expect
+        }
+        @Suppress("UNUSED_PARAMETER")
+        private fun hasSameBinaryContentAs(expect: Expect<Path>, targetPath: Path): Expect<Path> {
+            return expect
+        }
```

Comment:
that won't work; will cause the spec to fail. Delegate to the domain as you did in fluent, i.e.
```suggestion
        private fun hasSameTextualContentAs(expect: Expect<Path>, targetPath: Path, sourceCharset: Charset, targetCharset: Charset): Expect<Path> =
            expect.addAssertion(ExpectImpl.path.hasSameTextualContentAs(this, targetPath, sourceCharset, targetCharset))
        
        private fun hasSameBinaryContentAs(expect: Expect<Path>, targetPath: Path): Expect<Path> = 
            expect.addAssertion(ExpectImpl.path.hasSameBinaryContentAs(this, targetPath))
```